addons/refuel/script_component.hpp

```diff
@@ -30,4 +30,5 @@
     #define TIME_PROGRESSBAR(X) (X)
 #endif
 
-#define INTERACT_EXCEPTIONS "isNotInside", "isNotOnLadder", "isNotRefueling"
+#define INTERACT_EXCEPTIONS "isNotInside", "isNotOnLadder", "isNotRefueling", "isNotSwimming"
```

Comment:
maybe `#define INTERACT_EXCEPTIONS INTERACT_EXCEPTIONS_REFUELING, "isNotRefueling"`?

Reply:
Can improve, want to figure out this refuel thing first.